### DISCUSSION TITLE
Is there a way to add UNIX Timestamps to a project? If not, what is a good solution?

### DISCUSSION THREAD

**Discussion by Schofje** [community/community#49634] [Discussions]
I am organizing an open source community that will use the discussions feature on Github for our meeting notes. Is there something in particular that I'd have to do to ensure I can write down in the UNIX timestamp format in markdown?
perhaps `<t:TIMESTAMP>` seems to be an idea. Just like how Discord does it. It could help stimulating meetings if people know when they are in their own time.
It also helps with accessibility from my point of view since there is less time spent on converting times and more time on moving on to the actual thing that is happening on the time that is noted.

**Comment by ghost** [comment]
Yes, it is possible to add UNIX timestamps to a project, including in markdown files on GitHub. A UNIX timestamp is simply the number of seconds that have elapsed since January 1, 1970, so you can include this value in your markdown files using plain text or by using a template like <t:TIMESTAMP> as you suggested.

To include the timestamp in your markdown files, you can use a text editor that supports UNIX timestamps, such as Visual Studio Code or Sublime Text. Simply type the timestamp in plain text format or use the template, and save the file with a .md extension.

Another option is to use a third-party tool or plugin that automatically generates and inserts UNIX timestamps into your markdown files. There are several tools available for this purpose, including plugins for text editors and online services that can generate timestamps for you.

Including UNIX timestamps in your meeting notes can be a great way to keep track of time and ensure that everyone is on the same page. It can also help with accessibility and ensure that everyone knows what time the meeting is happening in their own time zone.

I hope this helps you add UNIX timestamps to your project and improve your meeting notes. Good luck with your open source community!

**Reply by Schofje** [reply]
I don't seem to see a timestamp based on someone's timezone in the github discussions when I type in 
<t:1678395988>
am I doing something wrong?
[test.md](https://github.com/community/community/files/10936066/test.md)

Even in a Markdown file

**Reply by ghost** [reply]
It's possible that the timestamp format you're using isn't being recognized by GitHub discussions. One thing you could try is to use a different timestamp format, such as the ISO 8601 format, which includes the date and time in a standardized way.

For example, the timestamp for March 10, 2023, at 10:00 AM Eastern Standard Time would be: 2023-03-10T10:00:00-05:00. You can include this format in your markdown file using plain text, or you can use a template like <t:2023-03-10T10:00:00-05:00> if you prefer.

If you're still having trouble, it may be worth reaching out to GitHub support or checking their documentation to see if there are any specific requirements or limitations for including timestamps in discussions.

I hope this helps! Let me know if you have any other questions or if there's anything else I can do to assist you.

**Comment by Abinashbunty** [comment]
Hi @ProtofrogUH  You can perhaps use this GitHub action to get timestamp. 😄 

[Get Timestamp action](https://github.com/marketplace/actions/get-timestamp-action)

**Reply by Schofje** [reply]
Unfortunately I'm more on the production side of things than the technical ones so I have less clue on how to use github actions.
I plan on using the discussions tab like a traditional forum and I see it uses YML which is different than the markdown github uses. How would the action work at all?
